skglm/solvers/group_bcd_solver.py

```diff
@@ -1,10 +1,10 @@
 import numpy as np
 from numba import njit
 
-from skglm.utils import check_group_compatible
+from skglm.utils import AndersonAcceleration, check_group_compatible
 
 
-def bcd_solver(X, y, datafit, penalty, w_init=None, p0=10,
+def bcd_solver(X, y, datafit, penalty, w_init=None, p0=10, use_acc=True, K=5,
```

Comment:
Last discussion: do we keep those exposed ? 
One of the claims of the paper is that K does not need to be tuned. Probably we always want to use AA. So we could make the API simpler by not exposing these 2 parameters.

Thoughts @QB3 @PABannier @Klopfe ?

Reply:
agreed.

Reply:
I agree too